### PR TITLE
Food and reagents in your hands and backpack no longer freeze while in space

### PR DESCRIPTION
**Pull request by DeityLink**
They may be allowed to freeze by dropping them directly onto a space turf still.

image

In more general terms, entropy is now proportional to the air pressure when said pressure is below 100kPa (so below room pressure basically), unless the item is directly placed on the floor. When pressure = 0 such as in space, this means that held items or ones in containers no longer experience entropy at all.

**Obviously this also applies to pills and other types of reagent containers, and whether they're in your pockets, backpack, belt, etc.**

:cl:
* tweak: Food and reagents in your hands and backpack no longer freeze while in space. They may be allowed to freeze by dropping them directly onto a space turf still. In more general terms, held items and ones in containers now experience reduced or no entropy in low pressure environments.

### CODE REVIEW

File: code/modules/reagents/reagent_containers.dm [hunk 1]
```diff
@@ -484,6 +484,9 @@ var/list/LOGGED_SPLASH_REAGENTS = list(FUEL, THERMITE)
 
 	var/diff = air.temperature - reagents.chem_temp
 
+	if (!isturf(loc) && (air.pressure < 100))//low pressure environments slow down entropy, unless the item is laid directly onto the floor so space meat remains frozen until brought in
+		diff *= air.pressure
```
Comment (west3436) [inline code/modules/reagents/reagent_containers.dm]: `diff *= air.pressure/100` to get a multiplier or is this intended?

Comment (DeityLink) [reply code/modules/reagents/reagent_containers.dm]: oh thank god you're here

**Comment by SonixApache** [comment]
what about belts, exosuits, webbings and other on-person containers

**Comment by DeityLink** [comment]
They will all protect your reagents from the cold of space now.

**Comment by RealestEstate** [comment]
I would prefer a slower temperature change when food/reagants are exposed to space, so as to become cold but at a much slower rate. Doesnt make sense that holding and putting in a backpack should effectively stasis it. Also heavily nerfs toolboxes' main gimmick.

**Comment by DeityLink** [comment]
It's not simple to introduce systemic changes that shake up the way players are used to play the game. The right way to do it is to add alongside a wealth of content that meshes with that system. Food getting cold in space would be less of an issue for example if players had either space-proof rations, or if spacesuits let you insert food in a way that keeps it insulated. But I'll leave that for someone more passionate about it to implement. In the meantime this PR addresses the more pressing issues.

**Pull request merged**